test/browser_tests/spec_suites/wagtail/wagtail-page-states-pages.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var DraftPage = require(
+  '../../page_objects/page_wagtail_states_pages.js' ).draftpage;
+var SharedPage = require(
+  '../../page_objects/page_wagtail_states_pages.js' ).sharedpage;
+var LivePage = require(
+  '../../page_objects/page_wagtail_states_pages.js' ).livepage;
+var LiveDraftPage = require(
+  '../../page_objects/page_wagtail_states_pages.js' ).livedraftpage;
+
+describe( 'Wagtail Draft Page', function() {
+  var page;
+
+  beforeAll( function() {
+    page = new DraftPage();
+    page.get();
+  } );
+
+  it( 'should not load in a browser',
+    function() {
+      expect( 404 );
+    }
+  );
+
+} );
+
+describe( 'Wagtail Shared Page', function() {
+  var page;
+
+  beforeAll( function() {
+    page = new SharedPage();
+  } );
+
+  it( 'should not load in a browser',
+    function() {
+      page.get();
+      expect( 404 );
+    }
+  );
+
+  it( 'should properly load in a browser',
+    function() {
+      page.getStaging();
+      expect( page.pageTitle() ).toContain(
+        'Shared Page | Consumer Financial Protection Bureau' );
```

Comment:
I grabbed the DB and it was still failing. When I ran the publish all shared pages scripts, it pased, but I think Shared Pages probably shouldn't be able to load in the browser until published, right?